OneSignalExample/Packages/com.onesignal.unity.core/Samples/OneSignaEexampleScene.unity

```diff
@@ -0,0 +1,260 @@
+%YAML 1.1
```

Comment:
This file name is misspelled, also `Samples/OneSignalExampleScene.unity` is already in this PR so I think we just need to delete this file.